auth/auth0/src/auth0Authentication.js

```diff
@@ -0,0 +1,104 @@
+const isomorphicFetch = require('isomorphic-fetch');
+const jwt = require('jsonwebtoken');
+const jwkRsa = require('jwks-rsa');
+const fromEvent = require('graphcool-lib').fromEvent;
+
+//Validates the request JWT token
+const verifyToken = token =>
+  new Promise(resolve => {
```

Comment:
Why not use async/await here too?

Reply:
Mainly because the jwt and jwks clients don't support promises.